src/command_modules/azure-cli-sqlvm/azure/cli/command_modules/sqlvm/_params.py

```diff
@@ -162,7 +162,7 @@ def load_arguments(self, _):
     with self.argument_context('sql vm', arg_group='SQL Server License') as c:
         c.argument('sql_server_license_type',
                    help='SQL Server license type.',
-                   options_list=['--license-type'],
+                   options_list=['--license-type', '-b'],
```

Comment:
Short options should be used sparingly, and there should be a clear connection between the short and long option. `-b` has no obvious connection to `--license-type`.